Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -55,10 +55,15 @@ OPTIONS:
                                 * in a (s)tream mode waits for input from STDIN,
                                   sending data to Virtuoso;
                                   messages are to be delimited by <delimiter>.
-  -d, --delimiter <delimiter>   Specifies the delimiter between sets of input
+  -b, --batch {e[abled]|d[isabled]} Specifies batch-mode: (e)nabled|(d)isabled.
+  -B, --eob <EOB>               Specifies the delimiter between sets of input
                                 data in the stream mode.
                                 Default: \n
-                                Kafka-style: \0
+                                Kafka-style: \x17
```

Comment:
I would say that as "batch mode" is formalized and properly defined, there\`s no such thing as "Kafka style" any longer. Kafka style was informal description of "batch mode", as Kafka-supervised processing was the only case when batch mode was used.

Reply:
@mgolosova 

Got it. I suppose, it would be better to represent it as:

`Dafault: \x17
No batch-mode: \n`